App_code/SAAO/User.cs

```diff
@@ -166,6 +183,7 @@ public User(string username)
             _mail = dr["mail"].ToString();
             _phone = dr["phone"].ToString();
             _wechat = dr["wechat"].ToString();
+            _filepush = (int)dr["FilePush"];

```

Comment:
reader.GetInt32(reader.GetOrdinal(columnName))
or

    Convert.ToInt32(reader[columnName])

is recommanded.